app/templates/views/service-settings.html

```diff
@@ -95,14 +95,15 @@ <h2 class="heading-small p-0 m-0">{{ _('Your service is in trial mode') }}</h2>
           }}
         {% endcall %}
 
-        {% call row() %}
-        
-          {% set txt = _('Daily message limit') %}
-          {{ text_field(txt)}}
-          {% set message_limit = _('{} notifications').format(current_service.message_limit | format_number) %}
-          {{ text_field(message_limit) }}
-          {{ text_field('') }}
-        {% endcall %}
+        {%if config["FF_EMAIL_DAILY_LIMIT"] == false %}
```

Comment:
```suggestion
        { %if config["FF_EMAIL_DAILY_LIMIT"] == false %}
```
Just nit picking formatting a little.